src/bootstrap/typescript.ts

```diff
@@ -0,0 +1,44 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import ts from 'typescript';
+import pathe from 'pathe';
+
+function printDiagnostics(diagnostics: readonly ts.Diagnostic[]): void {
```

Comment:
Official docs for API used in this file: https://github.com/microsoft/TypeScript/wiki/Using-the-Compiler-API